src/Entity/OgRole.php

```diff
@@ -243,6 +252,13 @@ public function delete() {
   }
 
   /**
+   * {@inheritdoc}
+   */
+  public function isDefaultRole() {
+    return in_array($this->getName(), [self::ANONYMOUS, self::AUTHENTICATED]);
```

Comment:
ah, but then this does not throw an exception.
Took some more reading the code to clear that up. How do we distinguish the default roles provided by `OgRoleInterface` from "default roles that will throw an exception when deleted"?